$NetBSD: patch-configure.ac,v 1.1 2020/09/17 19:43:47 schmonz Exp $

On macOS with clang 12, get explicit definitions of snprintf and
vsnprintf to avoid warnings (treated as errors) and thereby pass
configure.

--- configure.ac.orig	2015-06-18 11:33:58.000000000 +0000
+++ configure.ac
@@ -78,6 +78,7 @@ fi
 AC_CACHE_CHECK([for $snfunc and $vsnfunc],
                [kpse_cv_have_snfuncs],
                [AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <stdarg.h>
+                                                  #include <stdio.h>
                                                   char buf[16];
                                                   va_list ap;]],
                                                 [[$snfunc (buf, 16, "%s", "abc");
@@ -87,7 +88,8 @@ AC_CACHE_CHECK([for $snfunc and $vsnfunc
 if test "x$kpse_cv_have_snfuncs" != xyes; then
   AC_MSG_ERROR([Sorry, you need $snfunc and $vsnfunc.])
 fi
-AC_RUN_IFELSE([AC_LANG_PROGRAM([[char buf[4] = "abc";]],
+AC_RUN_IFELSE([AC_LANG_PROGRAM([[#include <stdio.h>
+                               char buf[4] = "abc";]],
                                [[if ($snfunc (buf, 1, "%s", "x") == 0 || buf[1] != 'b')
                                    return 1;]])],
               [],
